addons/hr_timesheet/models/hr_employee.py

```diff
@@ -2,13 +2,27 @@
 
 from ast import literal_eval
 
-from odoo import models, _
+from odoo import models, fields, _
 from odoo.exceptions import UserError
 
 
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
+    employee_has_timesheet = fields.Boolean(compute='_compute_employee_has_timesheet')
```

Comment:
A bit of nitpicking :innocent: 
As we are on the `employee` model, I think the `employee_` part can be removed from the field names, what do you think?

Reply:
True true :innocent: